docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.7
+
+# Better movement with [Ctrl + →] and [Ctrl + ←]
+ENV INPUTRC=/etc/inputrc
+
+RUN set -ex \
+  && echo '"\e[1;5C": forward-word' >> /etc/inputrc \
+  && echo '"\e[1;5D": backward-word' >> /etc/inputrc
+
+# Set working directory.
+# Mount your project under the same path.
+WORKDIR /usr/src
+
```

Comment:
Using `/usr/src` seems pretty strange. I think it is used in some systems (e.g. slackware?) to store actual sources.

Reply:
It could be whatever you want, I've seen some Dockerfiles using `/usr/src` since it is expected that you'll only put one source into it.